src/Compilers/Core/Portable/Emit/CommonPEModuleBuilder.cs

```diff
@@ -390,7 +457,7 @@ public Cci.IAssemblyReference GetContainingAssembly(EmitContext context)
         /// </summary>
         public string[] CopyStrings()
         {
-            return _stringsInILMap.CopyItems();
+            return _stringsInILMap.Value.CopyValues();
```

Comment:
>Value.CopyValues() [](http://example.com/codeflow?start=35&length=18)

This is not new to this PR, but does this mean we have two copies of all the strings from the module being compiled, all as strings?